test/config-next/va.json

```diff
@@ -30,7 +30,17 @@
     "features": {
       "GoogleSafeBrowsingV4": true,
       "IPv6First": true
-    }
+    },
+    "remoteVAs": [
+      {
+        "serverAddresses": ["va.boulder:19097"],
```

Comment:
The va-remote-a.json port is [9097](https://github.com/letsencrypt/boulder/pull/2802/files#diff-23e7c43ae539fbf350e22423cac8922aR19) and va-remote-b.json port is [9098](https://github.com/letsencrypt/boulder/pull/2802/files#diff-c58ba48e1e6a718b88d8d62697a89cd0R19). Should these be `19097` and `19098` with the `1` prefix or is that an error?